### PR TITLE
Localkubepodupdates

### PR DESCRIPTION
**Pull request by astronautyates**
Updated local kubernetes connection instructions to use connections UI instead of a lone connection file in the include directory to better align the local development experience with how users run Kubernetes in production/remote environment. All credit goes to @ReadytoRocc for coming up with this format, think it will save users a ton of time and confusion when running KPO locally as I have received feedback from customers that the current implementation is confusing.

### CODE REVIEW

**Comment by kentdanas** [comment]
@ReadytoRocc you mind double checking the updates here? Assuming this is technically correct this looks good to me.

File: code-samples/dags/kubepod-operator/kpo_example_1.py [hunk 1]
```diff
@@ -1,42 +1,20 @@
...
+    example_kpo = KubernetesPodOperator(
+        kubernetes_conn_id="k8s_conn",
         image="hello-world",
-        labels={"<pod-label>": "<label-name>"},
         name="airflow-test-pod",
         task_id="task-one",
-        in_cluster=in_cluster,  # if set to true, will look in the cluster, if false, looks for file
-        cluster_context="docker-desktop",  # is ignored when in_cluster is set to True
-        config_file=config_file,
+        cluster_context="docker-desktop",
```
[the remark this answers is not in the document]
Comment (TJaniF) [reply code-samples/dags/kubepod-operator/kpo_example_1.py:15]: I tested and confirmed :) Will remove that parameter. Thanks! (and sorry it took ages)

Comment (TJaniF) [inline code-samples/dags/kubepod-operator/kpo_example_1.py:15]:
```suggestion
```

File: code-samples/dags/kubepod-operator/kpo_example_1.py [hunk 2]
```diff
@@ -1,42 +1,19 @@
...
-namespace = conf.get("kubernetes", "NAMESPACE")
-# This will detect the default namespace locally and read the
-# environment namespace when deployed to Astronomer.
-if namespace == "default":
-    config_file = "/usr/local/airflow/include/.kube/config"
-    in_cluster = False
-else:
-    in_cluster = True
-    config_file = None
+from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
```
Comment (TJaniF) [inline code-samples/dags/kubepod-operator/kpo_example_1.py:3]:
```suggestion
from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import (
    KubernetesPodOperator,
)
```

**Review by TJaniF — APPROVED** [review]
Tested and works great! Sorry it took so long @astronautyates 😅

**Pull request merged**